app/(header-default)/compte/page.tsx

```diff
@@ -0,0 +1,71 @@
+import { Badge } from '#components-ui/badge';
+import { FullTable } from '#components/table/full';
+import { AppScope, hasRights } from '#models/user/rights';
+import { getAbsoluteSiteUrl } from '#utils/server-side-helper/app/get-absolute-site-url';
+import getSession from '#utils/server-side-helper/app/get-session';
+import { Metadata } from 'next';
+import { redirect } from 'next/navigation';
+
+export const metadata: Metadata = {
+  title: 'Votre compte utilisateur de l’Annuaire des Entreprises',
+  alternates: {
+    canonical: 'https://annuaire-entreprises.data.gouv.fr/compte',
+  },
+  robots: 'noindex, nofollow',
+};
+
+const AccountPage = async () => {
+  const session = await getSession();
+
+  if (!hasRights(session, AppScope.isAgent)) {
+    return redirect(getAbsoluteSiteUrl('/lp/agent-public'));
```

Comment:
```suggestion
    return redirect('/lp/agent-public');
```

As we are in a page, the getAbsoluteSiteUrl can be omitted ?